torch/csrc/distributed/rpc/python_functions.cpp

```diff
@@ -37,6 +41,20 @@ std::shared_ptr<FutureMessage> py_rpc_builtin(
         Stack stack = torch::jit::createStackForSchema(
             op->schema(), args, kwargs, c10::nullopt);
 
+        auto& autogradContainer = DistAutogradContainer::getInstance();
+        if (autogradContainer.hasValidContext()) {
+          // Get all the ivalues for the operator.
+          auto ivalues =
+              torch::jit::last(stack, op->schema().arguments().size());
+
+          // Attach the appropriate autograd edges.
+          auto grad_fn = addSendRpcBackward(ivalues);
```

Comment:
How does this work for `RRef` args?

Reply:
We discussed this offline, as part of .to_here() we would attach the necessary args.